docs/moments.rst

```diff
@@ -19,14 +19,35 @@ axis::
     >>> moment_0_along_x = cube.moment(order=0, axis=2)  # doctest: +SKIP
 
 .. note:: These follow the mathematical definition of moments, so the second
-          moment is computed as the variance. For linewidth maps, see the
+          moment is computed as the variance. For the actual formulas used for
+          the moments, please see `the relevant documentation here 
+          <https://spectral-cube.readthedocs.io/en/latest/api/spectral_cube.SpectralCube.html#spectral_cube.SpectralCube.moment>`_.
+          For linewidth maps, see the
           `Linewidth maps`_ section.
+          
+You may also want to convert the unit of the datacube into a velocity one before
+you can obtain a genuine velocity map via a 1st moment map. So first it will be necessary to 
+apply the :class:`~spectral_cube.SpectralCube.with_spectral_unit` method from this package with the proper attribute settings::
+
+    >>> nii_cube = cube.with_spectral_unit(u.km/u.s, velocity_convention='optical', rest_value=6584*u.AA)  # doctest: +SKIP
+
+Note that the `rest_value` in the above code refers to the wavelength of the targeted line 
```

Comment:
this should be in double-backticks because it's a variable name.  I think this is the last small fix needed to get the tests to pass